### DISCUSSION TITLE
docker image not found!

### DISCUSSION THREAD

**Discussion by capitangiaco** [akvorado/akvorado#696] [Q&A]
docker-compose up -d
Pulling akvorado-conntrack-fixer (ghcr.io/akvorado/akvorado:1.8.3)...
Trying to pull repository ghcr.io/akvorado/akvorado ...
Pulling repository ghcr.io/akvorado/akvorado
ERROR: Error: image akvorado/akvorado:1.8.3 not found

**Comment by Anaximander-Thales** [comment]
I'm encountering this as well.

It appears that you need to pass the docker folder in the command

in my case, podman compose -f ./docker/docker-compose.yml up -d.  I believe it's the same command for docker.

**Reply by vincentbernat** [reply]
There is a .env file that should be used. Just this file is not enough to bring everything up.

**Reply by Anaximander-Thales** [reply]
Oh! Good to know, I'm learning podman/docker so, this is proving quite the effort.

So, looking at the the .env file,  (I'm adding notes to the right)
```
COMPOSE_PROJECT_NAME=akvorado                 <--- creating a project name for ALL the podman containers created
COMPOSE_FILE=docker/docker-compose.yml       <--- this should be pointing podman/docker to find this file

## GeoIP provider.
# Option 1: IPInfo
COMPOSE_FILE=${COMPOSE_FILE}:docker/docker-compose-ipinfo.yml           <----  including compose-file list
# Option 2: Maxmind
# COMPOSE_FILE=${COMPOSE_FILE}:docker/docker-compose-maxmind.yml   <----  including compose-file list
# GEOIPUPDATE_ACCOUNT_ID=...
# GEOIPUPDATE_LICENSE_KEY=...

## Monitoring
# COMPOSE_FILE=${COMPOSE_FILE}:docker/docker-compose-monitoring.yml

## Grafana
# COMPOSE_FILE=${COMPOSE_FILE}:docker/docker-compose-grafana.yml

## Demo
COMPOSE_FILE=${COMPOSE_FILE}:docker/docker-compose-demo.yml           <---- including compose-file list, there for demo purposes

## For testing only: use a ClickHouse cluster
# COMPOSE_FILE=${COMPOSE_FILE}:docker/docker-compose-clickhouse-cluster.yml


```
So, it appears the .env contains the path, but for some reason is not being used in my case (and apparently this OP case as well from a year ago).

Do I need to source this file? Or is there a configuration file that I need to update to see this file?

**Reply by vincentbernat** [reply]
Docker automatically uses this file. I don't know for Podman. Are you sure you get the same error as the original poster? This does not seem related.

**Reply by Anaximander-Thales** [reply]
I was before, but after sourcing it the .env file (to test, which still fails with podman), I get pass files with -f, which is not in the original error. So, either something changed (doubt), or I really did not pay attention, which is quite possible with reading text for the last several hours.

That said, the .env file is a good way to investigate how to get podman to use it, or how to pass podman the information that's in the .env file. I have been assuming that podman used the .env file, but you just taught me that if I'm passing the file, I'm not using it. thank you.
